pkg/psmdb/pmm.go

```diff
@@ -1,6 +1,8 @@
 package psmdb
 
 import (
+	"fmt"
```

Comment:
We are trying to use `errors` package for errors instead of `fmt`

Reply:
done 6f79105